apps/client/src/app/debug/chat.tsx

```diff
@@ -41,6 +41,11 @@ const ChatPage = ({
     }
   };
 
+  const sendStart = async (e: React.MouseEvent<HTMLButtonElement>) => {
+    e.preventDefault();
+    await socket.emit('start_game', roomId);
```

Comment:
A QUESTION: Does `await` needed in `socket.emit` event?

Reply:
I don't know much about async and await. I tried deleting them, and it worked fine. So, no need krub. 😂